### PR TITLE
Updated run script to ignore yarn

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes
Added ability to add `--no-yarn` to ignore the yarn build

## Related issue
Link to the GitHub issue this pull request addresses using the `#issuenum` format.  If it completes an issue, use `Fixes #issuenum` to automatically close the issue.

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [x] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
* [ ] Tests added (or description of how to test) for any new features.
* [ ] User documentation updated for UI or technical changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ Changes
 
 22.01 to 22.05
 --------------
+* [Developer]: Add `--no-yarn` to skip the yarn build in `run.sh`
```
Comment (ericenns) [inline CHANGELOG.md:6]: Need to update this to mention `--prod` option.

File: run.sh [hunk 2]
```diff
@@ -1,13 +1,30 @@
...
+  do
+    if [ "$1" = "--create-db" ];
+    then
+       COMMANDS+=("-Dspring-boot.run.arguments=\"--hbm.dev.auto=create\"")
+       ADD_DB=true
+       echo "Dropping then Creating/Recreating database schema"
+    elif [ "$1" = "--no-yarn" ];
+    then
+      COMMANDS+=("-Dskip.yarn")
+      elif [ "$1" = "--prod" ];
```
Comment (ericenns) [inline run.sh:18]: Can you fix this indentation, so it is consistent.

**Review by ericenns — CHANGES_REQUESTED** [review]
Can you also update `doc/developer/getting-started/index.md` to mention the options in run.sh

File: run.sh [hunk 3]
```diff
@@ -1,13 +1,30 @@
...
+
+if [ "$ADD_DB" = false ]; then
    echo "Updating database schema without dropping"
-  fi
+fi
 
-mvn clean spring-boot:run -Dspring-boot.run.arguments="--spring.profiles.active=dev ${ADD}"
\ No newline at end of file
+ADD="$(printf "%s " "${COMMANDS[@]}" )"
+mvn clean spring-boot:run -Dspring-boot.run.profile=${PROFILE} $ADD
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply run.sh]: Donw

File: CHANGELOG.md (hunk 1 above)
Comment (joshsadam) [reply CHANGELOG.md]: Updated in 60965d0

**Review by ericenns — APPROVED** [review]
👍 Looks good to me

**Pull request merged**